components/raftstore-v2/tests/integrations/test_conf_change.rs

```diff
@@ -178,3 +178,89 @@ fn test_remove_by_conf_change() {
     assert_eq!(region_state.get_state(), PeerState::Tombstone);
     assert_eq!(raft_engine.get_raft_state(region_id).unwrap(), None);
 }
+
+#[test]
+fn test_tombstore() {
```

Comment:
```suggestion
fn test_tombstone() {
```

Should mention what it's testing and what problem it can encouter.

Reply:
fixed